boards/hardkernel/odroid_go/odroid_go_appcpu.dts

```diff
@@ -40,27 +40,28 @@
 			read-only;
 		};
 
-		/* Reserve 1024kB for the application in slot 0 */
+		/* Reserve 5120kB for the application in slot 0 */
```

Comment:
Hey, the partitions for the APPCPU should avoid using the same address space the PROCPU is using, therefore current layout doesn't make sense. It's better to leave it as is as it needs to be fixed.